flows/engine/services.go

```diff
@@ -7,17 +7,22 @@ import (
 // WebhookServiceFactory resolves a session to a webhook service
 type WebhookServiceFactory func(flows.Session) flows.WebhookService
 
+// NLUServiceFactory resolves a session and classifier to an NLU service
+type NLUServiceFactory func(flows.Session, *flows.Classifier) flows.NLUService
```

Comment:
Sorry, I'm still confused on this naming.

The action is called `call_classifier` but goflow is calling the NLUService for that? Should this just be `ClassifierService`? Seems that would be more consistent.

Reply:
To me it feels consistent with say, the "airtime" service can "transfer" for now.. and might do other stuff in the future. But I guess it's not quite the same because the NLU stuff is tied to particular classifier... I'll change it...